filter/etc/e-smith/templates/etc/rspamd/rspamd.local.lua/10base

```diff
@@ -0,0 +1,162 @@
+{
+
+if  ( $rspamd{'SpamCheckStatus'} eq 'enabled'){
+$OUT .= ' -- symbols/lua rules are not disabled';
+}
+
+else {
+    # Disable all lua rules
+
+$OUT .= q(
+-- disable symbol rules
+rspamd_config:add_condition('vvhkjcvhkcxhvFWD_GOOGLE', function(task) return false end)
```

Comment:
This list is way too long and very hard to maintain.

I propose 2 things:
- disable only relevant symbols, so in this just `SPOOF_REPLYTO`
- rewrite the code below using a loop (if possible)

Loop example (not tested):
```lua
bad_symbols={'SPOOF_REPLYTO', 'FORWARDED'};
for key,value in ipairs(bad_symbols) 
do
   rspamd_config:add_condition(value, function(task) return false end)
end
```

Reply:
This list comes from the read of logs, you can in logs have  many little symbol with little weigh like the headers are (0.10,0.005...) and from others that have a 0.00 score.

IIUC 

/usr/share/rspamd/plugins/*.lua are the lua part of modules
/usr/share/rspamd/rules/*.lua are the lua part of rules and something called virtual-symbol (a symbol without rule)

in short if we do not match all this rule, we could be sure that a new ticket will come....hey guy I have disabled rspamd and I still have zero score or some score weigh in my maillog....why ?

Reply:
I get your point, but still I like a more restrict list and the cost of a couple of support ticket.

What do you think @DavidePrincipi @filippocarletti ?